File: src/main/java/com/github/sorusclient/client/module/impl/fullbright/v1_8_9/FullBrightHook.java
package com.github.sorusclient.client.module.impl.fullbright.v1_8_9;

import com.github.sorusclient.client.Sorus;
import com.github.sorusclient.client.module.ModuleManager;
import com.github.sorusclient.client.module.impl.fullbright.FullBright;

public class FullBrightHook {

    public static float modifyGamma(float gamma) {
        FullBright fullBright = Sorus.getInstance().get(ModuleManager.class).get(FullBright.class);
        if (fullBright.isEnabled()) {
            return 50f;
        } else {
            return gamma;
        }
    }

}

File: src/main/kotlin/com/github/sorusclient/client/module/impl/fullbright/v1_8_9/FullBrightHook.kt
package com.github.sorusclient.client.module.impl.fullbright.v1_8_9

import com.github.sorusclient.client.module.ModuleManager
import com.github.sorusclient.client.module.impl.fullbright.FullBright

object FullBrightHook {

    @JvmStatic
    fun modifyGamma(gamma: Float): Float {
        val fullBright = ModuleManager.get<FullBright>()
        return if (fullBright.isEnabled()) {
            50f
        } else {
            gamma
        }
    }

}